mnefun/_mnefun.py

```diff
@@ -1818,7 +1817,11 @@ def do_preprocessing_combined(p, subjects, run_indices):
             n_jobs=p.n_jobs_fir, n_jobs_resample=p.n_jobs_resample,
             projs=projs, bad_file=bad_file, disp_files=p.disp_files,
             method='fft', filter_length=p.filter_length, force_bads=False,
-            l_trans=p.hp_trans, h_trans=p.lp_trans)
+            l_trans=p.hp_trans, h_trans=p.lp_trans, eeg_ref=False)
+        for ii in raw_orig.info['projs']:
+            if ii['desc'] == 'Average EEG reference':
+                eeg_ref_proj = ii
+                raw_orig.add_proj(eeg_ref_proj).apply_proj()
 
```

Comment:
I would probably do:
```
raw.remove_proj(ii)
http://mne-tools.github.io/stable/generated/mne.io.Raw.html#mne.io.Raw.add_eeg_average_proj
```
and `proj` is a better name for the thing to iterate instead of `ii`, which is usually used for an index

But really this probably isn't the right place for this, but rather where we set the bad channels from the `bads/*.txt` file